### PR TITLE
update embeddings for base_uae_mem

### PR DESCRIPTION
**Pull request by chejennifer**
sv diffs between old base_uae_mem and new base_uae_mem for dc queries: https://storage.mtls.cloud.google.com/datcom-embedding-diffs/chejennifer_base_uae_mem_2024_06_11_11_22_49.html 

sv diffs between medium_ft and new base_uae_mem for bard queries: https://storage.mtls.cloud.google.com/datcom-embedding-diffs/chejennifer_medium_ft_2024_06_11_10_40_21.html

### CODE REVIEW

File: tools/nl/embeddings/data/preindex/medium/sv_descriptions.csv [hunk 1]
```diff
@@ -1686,7 +1686,7 @@ Count_HousingUnit_IncompletePlumbingFacilities_OccupiedHousingUnit,Number of Occ
 Count_HousingUnit_NoCashRent,Number of Renter-Occupied Housing Units with No Cash Rent
 Count_HousingUnit_OccupiedHousingUnit,Number of Occupied Housing Units
 Count_HousingUnit_OwnerOccupied,Number of Owner-Occupied Housing Units
-Count_HousingUnit_RenterOccupied,Number of Renter-Occupied Housing Units
```
Comment (pradh) [inline tools/nl/embeddings/data/preindex/medium/sv_descriptions.csv:1689]: For future changes, per Bo's observations, I think we should use terminology consistently.  e.g., other occurrences of "Renter-occupied Housing units" should also be "rented houses".  Have a TODO?

**Review by pradh — APPROVED** [review]
Thanks!  Please wait for Bo's review too.

File: tools/nl/embeddings/data/preindex/medium/sv_descriptions.csv (hunk 1 above)
Comment (shifucun) [reply tools/nl/embeddings/data/preindex/medium/sv_descriptions.csv:1689]: +1 for TODO. And also there are owner occupied houses we want to clean up too.

**Review by shifucun — APPROVED** [review]

File: tools/nl/embeddings/data/preindex/medium/sv_descriptions.csv (hunk 1 above)
Comment (chejennifer) [reply tools/nl/embeddings/data/preindex/medium/sv_descriptions.csv:1689]: don't think I can leave comments in csv files, so added to buganizer: https://b.corp.google.com/issues/346618949

**Comment by chejennifer** [comment]
thanks for the reviews!

**Pull request merged**